src/Config.php

```diff
@@ -1205,11 +1206,11 @@ public function processLongArgument($arg, $pos)
                     $eqPos = strpos($arg, '=');
                     try {
                         if ($eqPos === false) {
-                            $this->values[$arg] = $arg;
+                            $this->unknown[$arg] = $arg;
```

Comment:
It's a good thing to do generally, but I think this PR will be delayed, since there should be tests for `Config.php` (there is no tests).

Also it will be good idea to have code coverage for whole PHP_CodeSniffer